junebug/api.py

```diff
@@ -301,3 +304,33 @@ def get_message_status(self, request, channel_id, message_id):
     @app.route('/health', methods=['GET'])
     def health_status(self, request):
         return response(request, 'health ok', {})
+
+    @app.route('/channels_health', methods=['GET'])
+    @inlineCallbacks
+    def channels_health(self, request):
+        channels_stuck = []
+
+        base_url = "http://%s:%s/api/queues/%s" % (
+            self.amqp_config['hostname'],
+            self.amqp_config.get('api_port', '15672'),
```

Comment:
So by default on rabbitmq, `5672` is the main port, and `15672` is the management port. So if I'm specifying my rabbitmq, I would usually specify a port of `5672`, and then this code will try to get management things from the incorrect port.

We will need a separate config value for the rabbitmq management interface port.

Reply:
`api_port` is a new config value for when your management is on something other than `15672`. `port` is the existing one. 

I'll document appropriately so it makes more sense.

Reply:
Maybe we should then name it `management_port` or something like that, so it's more obvious?